### PR TITLE
Create runbooks for seven VirtOperator related alerts.

### PR DESCRIPTION
**Pull request by yuhaohaoyu**
Signed-off-by: Hao Yu <yuh@us.ibm.com>

### CODE REVIEW

File: docs/runbooks/LowReadyVirtOperatorsCount.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# LowReadyVirtOperatorCount
```
Comment (enp0s3) [inline docs/runbooks/LowReadyVirtOperatorsCount.md:1]: @yuhaohaoyu  I can't find this alert in the Kubevirt codebase. Is that a deprecated alert?

Comment (yuhaohaoyu) [reply docs/runbooks/LowReadyVirtOperatorsCount.md:1]: Good catch. Its 'LowReadyVirtOperatorsCount', there is an 's'.

Comment (yuhaohaoyu) [reply docs/runbooks/LowReadyVirtOperatorsCount.md:1]: @enp0s3 
Hi Igor, 

I fixed the type, great catch. Can you please go review the pull 45, https://github.com/kubevirt/monitoring/pull/45

It contains these changes. I will close this pull later. Sorry for the confusion.

Comment (enp0s3) [reply docs/runbooks/LowReadyVirtOperatorsCount.md:1]: @yuhaohaoyu Sure np!

**Comment by djfjeff** [comment]
To make this runbook compatible with both upstream and downstream, would you consider exporting the namespace and using it after (for example, kubevirt for upstream, openshift-cnv for downstream)? I see in other runbook the following :

- Set the environment variable NAMESPACE

```
export NAMESPACE="$(kubectl get kubevirt -A -o custom-columns="":.metadata.namespace)"
```

- Then use it in other commands, for example

`kubectl -n $NAMESPACE get pods -l kubevirt.io=virt-api`

File: docs/runbooks/LowReadyVirtOperatorsCount.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# LowReadyVirtOperatorCount 
+
+## Meaning
+
+The virt-operator is the first k8s operator coming to alive in a KubeVirt cluster. With the primary responsible for the installation, live-update, live-upgrade of a KubeVirt cluster, it monitors the life-cycle of top-level controllers such as virt-controller, virt-handler, virt-launcher, etc. and manages their reconciliation. In addition, virt-operator is responsible for cluster-wide tasks such as certificate rotation and some infrastructure management, etc.
```
Comment (sradco) [inline docs/runbooks/LowReadyVirtOperatorsCount.md:5]: @yuhaohaoyu Please update same as https://github.com/kubevirt/monitoring/pull/45

Comment (sradco) [reply docs/runbooks/LowReadyVirtOperatorsCount.md:5]: @yuhaohaoyu ping

**Comment by sradco** [comment]
> To make this runbook compatible with both upstream and downstream, would you consider exporting the namespace and using it after (for example, kubevirt for upstream, openshift-cnv for downstream)? I see in other runbook the following :
> 
> * Set the environment variable NAMESPACE
> 
> ```
> export NAMESPACE="$(kubectl get kubevirt -A -o custom-columns="":.metadata.namespace)"
> ```
> 
> * Then use it in other commands, for example
> 
> `kubectl -n $NAMESPACE get pods -l kubevirt.io=virt-api`

@yuhaohaoyu Can you please add the step
* Set the environment variable NAMESPACE
 
 ```
 export NAMESPACE="$(kubectl get kubevirt -A -o custom-columns="":.metadata.namespace)"
 ```
 
 and replace "-n kubevirt" with "-n $NAMESPACE" in all commands? ?

**Comment by sradco** [comment]
> > To make this runbook compatible with both upstream and downstream, would you consider exporting the namespace and using it after (for example, kubevirt for upstream, openshift-cnv for downstream)? I see in other runbook the following :
> > 
> > * Set the environment variable NAMESPACE
> > 
> > ```
> > export NAMESPACE="$(kubectl get kubevirt -A -o custom-columns="":.metadata.namespace)"
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > 
> > * Then use it in other commands, for example
> > 
> > `kubectl -n $NAMESPACE get pods -l kubevirt.io=virt-api`
> 
> @yuhaohaoyu Can you please add the step
> 
> * Set the environment variable NAMESPACE
> 
> ```
> export NAMESPACE="$(kubectl get kubevirt -A -o custom-columns="":.metadata.namespace)"
> ```
> 
> and replace "-n kubevirt" with "-n $NAMESPACE" in all commands? ?

@yuhaohaoyu Please update this PR based on the comments

**Comment by sradco** [comment]
@yuhaohaoyu Hi, Please fixe based on the comments above and rebase...

**Comment by yuhaohaoyu** [comment]
> @yuhaohaoyu Hi, Please fixe based on the comments above and rebase...

Hi Shirley, This PR was replaced by #45 that has merged. Do you want me to close this PR ?

**Pull request closed without merging**